apps/web/src/lib/transcription-service.ts

```
@@ -54,80 +54,87 @@ export async function fetchTranscript({
-      metadata = await fetchYouTubeMetadata(url);
-    } catch {
-      console.log('YouTube metadata fetch failed, continuing without');
-    }
-  }
+              if (fullText.length > 50) {
+                return {
+                  success: true,
+                  transcript: fullText,
+                  segments,
+                  source: 'youtube',
+                  wordCount: fullText.split(/\s+/).length,
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)